app/views/admin/emails/index.html.haml

```diff
@@ -100,6 +102,26 @@
                 'data-body-text' => "Dear {name},\n\nThe Conference Call for Papers Details of {conference} has changed.\nNew Dates : {cfp_start_date} - {cfp_end_date}.\n Link to Schedule {schedule_link} \n\nBest wishes\n\n{conference} Team" } Load Template
                 %a.btn.btn-link.control_label.template_help_link{ 'data-name' => 'updated_cfp_help' } Show Help
                 = render partial: 'help', locals: {id: 'updated_cfp_help', show_event_variables: false}
+              #booth.tab-pane{ role: 'tabpanel' }
+                = f.input :send_on_booths_acceptance
+                = f.input :booths_acceptance_subject
+                = f.input :booths_acceptance_body, input_html: { rows:10, cols: 20 }
+                %a.btn.btn-link.control_label.load_template{ 'data-subject-input-id' => 'email_settings_booths_acceptance_subject',
+                'data-subject-text' => 'Your booth has been accepted!',
+                'data-body-input-id' => 'email_settings_booths_acceptance_body',
+                'data-body-text' => "Dear {name},\n\nWe are really pleased to inform you that your booth request {booth_title} has been accepted for the conference {conference}.\nPlease click the confirm button to let us know you can make it as soon as possible!\n\nFeel free to contact us with any questions or concerns.\n\nWe look forward to seeing you there.\n\nBest wishes\n\n{conference} Team"} Load Template
```

Comment:
`We are pleased to inform you` (without the *really*)
`We are looking forward to seeing you` (the expression you are using should be written like this)